packages/regl-worldview/src/utils/commandUtils.js

```diff
@@ -98,6 +98,8 @@ export const blend = {
   },
 };
 
+export const defaultDepth = { enable: true, mask: true };
```

Comment:
if this value is only used in `triangles.js` it's probably better to colocate it w/ that file if/until it's needed in other places as well.

Reply:
Good suggestion!